modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -434,47 +409,37 @@ namespace gapi {
         }
 
         /**
-         * @brief Find a kernel (by its API), given the look-up order.
+         * @brief Find a kernel (by its API)
          *
-         * If order is empty, returns first suitable implementation.
+         * Returns first suitable implementation.
```

Comment:
"first suitable implementation" assumes there may be more than one suitable implementations.

Our new design forbids that.

Reply:
Fix